vonk/configuration/db_mongo.rst

```diff
@@ -8,6 +8,9 @@ Using MongoDB
 =============
 We assume you already have MongoDB installed. If not, please refer to the `MongoDB download <https://www.mongodb.com/download-center>`_ pages.
 
+Vonk can work with MongoDb 4.0 and higher. Since Vonk version 3.7.0 Vonk uses the MongoDb Aggregation Framework heavily, and you are advised to upgrade to MongoDb 4.4 (or newer). 
+In this version issue SERVER-xyz is solved. 
```

Comment:
Not sure what issue this is, perhaps link to Mongo docs?

Reply:
Yes, forgot to finish that.